tests/suites/test_suite_nist_kw.data

```diff
@@ -20,10 +20,6 @@ NIST KW init #4 AES-224: bad key size
 depends_on:MBEDTLS_AES_C
 mbedtls_nist_kw_setkey:MBEDTLS_CIPHER_ID_AES:224:1:MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA
 
-NIST KW init #5 BLOWFISH-128: bad cipher
```

Comment:
As Gilles pointed out for another test, I think we can keep this one by using 3DES instead of blowfish.